tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -645,12 +645,13 @@ static void btp_send_discovery_completed_ev(struct bt_conn *conn, uint8_t status
 }
 
 static void btp_send_pac_codec_found_ev(struct bt_conn *conn,
-					const struct bt_audio_codec_cfg *codec_cfg,
+					const struct bt_audio_codec_cap *codec_cap,
 					enum bt_audio_dir dir)
 {
 	struct btp_bap_codec_cap_found_ev ev;
 	struct bt_conn_info info;
 	const struct bt_audio_codec_data *data;
+	struct bt_audio_codec_cfg *codec_cfg = (struct bt_audio_codec_cfg *)codec_cap;
```

Comment:
This is a poor idea: The two struct do not represent the same data (and `bt_audio_codec_cfg_get_val` should not work with a `codec_cap`), and the two structs will not be equal in future commits. 

Since your first line of change is correct, i.e. this is in fact a codec capability, the usage of `codec_cap` in this function needs to be updated. 

Unfortunately we do not yet at this point have `bt_audio_codec_cap_get_X` functions to help retrieve specific values from a codec capability, so you will need to search for it manually.

Reply:
The documentation of `bt_audio_codec_cfg_get_val` should be updated then:
https://github.com/zephyrproject-rtos/zephyr/blob/66018d5f0a3ce4b817726701ae6cb14a483cf5e0/include/zephyr/bluetooth/audio/audio.h#L641-L657

Reply:
Ah, I clearly missed that one - Will post a PR to update that